thunder/benchmarks/neva.py

```diff
@@ -0,0 +1,6275 @@
+            )
+            l_self_modules_vision_encoder_modules_vision_model_modules_encoder_modules_layers_modules_8_modules_self_attn_modules_q_proj_parameters_weight_ = (
+                l_self_modules_vision_encoder_modules_vision_model_modules_encoder_modules_layers_modules_8_modules_self_attn_modules_q_proj_parameters_bias_
+            ) = None
+            query_states_16 = linear_48 * 0.125
+            linear_48 = None
+            linear_49 = torch._C._nn.linear(
+                hidden_states_57,
+                l_self_modules_vision_encoder_modules_vision_model_modules_encoder_modules_layers_modules_8_modules_self_attn_modules_k_proj_parameters_weight_,
+                l_self_modules_vision_encoder_modules_vision_model_modules_encoder_modules_layers_modules_8_modules_self_attn_modules_k_proj_parameters_bias_,
+            )
+            l_self_modules_vision_encoder_modules_vision_model_modules_encoder_modules_layers_modules_8_modules_self_attn_modules_k_proj_parameters_weight_ = (
+                l_self_modules_vision_encoder_modules_vision_model_modules_encoder_modules_layers_modules_8_modules_self_attn_modules_k_proj_parameters_bias_
+            ) = None
+            view_56 = linear_49.view(2, -1, 16, 64)
+            linear_49 = None
+            transpose_41 = view_56.transpose(1, 2)
+            view_56 = None
+            key_states_16 = transpose_41.contiguous()
+            transpose_41 = None
+            linear_50 = torch._C._nn.linear(
+                hidden_states_57,
+                l_self_modules_vision_encoder_modules_vision_model_modules_encoder_modules_layers_modules_8_modules_self_attn_modules_v_proj_parameters_weight_,
+                l_self_modules_vision_encoder_modules_vision_model_modules_encoder_modules_layers_modules_8_modules_self_attn_modules_v_proj_parameters_bias_,
+            )
+            hidden_states_57 = (
+                l_self_modules_vision_encoder_modules_vision_model_modules_encoder_modules_layers_modules_8_modules_self_attn_modules_v_proj_parameters_weight_
+            ) = (
+                l_self_modules_vision_encoder_modules_vision_model_modules_encoder_modules_layers_modules_8_modules_self_attn_modules_v_proj_parameters_bias_
+            ) = None
+            view_57 = linear_50.view(2, -1, 16, 64)
+            linear_50 = None
+            transpose_42 = view_57.transpose(1, 2)
+            view_57 = None
+            value_states_16 = transpose_42.contiguous()
+            transpose_42 = None
+            view_58 = query_states_16.view(2, 257, 16, 64)
+            query_states_16 = None
+            transpose_43 = view_58.transpose(1, 2)
+            view_58 = None
+            contiguous_26 = transpose_43.contiguous()
+            transpose_43 = None
+            query_states_17 = contiguous_26.view(32, -1, 64)
+            contiguous_26 = None
+            key_states_17 = k
```

Comment:
This code is unused and can be removed.

Reply:
GitHub is having trouble rendering correctly this large diff. This comment was referring to lines 6150-6153 with the pattern:
```py
function_ctx = torch.autograd.function.FunctionCtx()
function_ctx = None
```